tf2_yolov4/model.py

```diff
@@ -22,7 +22,7 @@ def YOLOv4(
     YOLOv4 Model
 
     Args:
-        input_shape (Tuple[int]): Input shape of the image
+        input_shape (Tuple[int]): Input shape of the image (H,W,C) . The Height and Width must be multiple of 32
```

Comment:
You may specify what exceptions could be raise in the docstring, see https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
cool!